students/KellyM/Lesson2/lesson 2 - series.py

```diff
@@ -0,0 +1,48 @@
+def fib(n):
+    """ This function calculates a Fibonacci Series, which is a series that
+        starts with 0 and 1, then calculates the next number by adding the
+        previous two numbers."""
+    if n == 1:
+        return 1
+    elif n==2:
```

Comment:
The Python Pep8 style guide prefers whitespace around operators. We recommend that you check your code against an online Pep8 linter, or configure your text editor to catch Pep8 violations as you type. There are instructions in the course materials for how to set this up for SublimeText, and the PyCharm IDE will provide this functionality by default.